compose.yaml

```diff
@@ -0,0 +1,15 @@
+services:
+  postgres_db:
+    container_name: "postgres_db"
+    image: "postgres"
```

Comment:
It is always recommended to specify the tag like "postgres:17"

Reply:
solved